plugin.video.raiyoyo/default.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+import xbmcaddon
+import xbmc
+import xbmcplugin
+import xbmcgui
+import os
+import sys
+import re 
+from resources.lib.raiyoyo import Raiyoyo
+from phate89lib import kodiutils, staticutils
+
+HANDLE=int(sys.argv[1])
+
+raiyoyo = Raiyoyo()
+raiyoyo.log = kodiutils.log
+
+def stamp_ep(ep):
+    ep['mediatype'] = 'video'
+    xbmc.log("STAMP_EP:")
+    xbmc.log(str(ep))
+    kodiutils.addListItem("[COLOR blue]"+ep["title"]+"[/COLOR]", {'mode':'playRaiyoyo', 'url': ep["url"]}, thumb=ep["thumbs"],videoInfo=ep,isFolder=False)
+
+def root():
+    kodiutils.addListItem("Programmi",{'mode':'elenco_video'})
+    kodiutils.endScript()
+
+def elenco_video_groupList():
+    for group in raiyoyo.get_url_groupList():
+      kodiutils.addListItem(group["title"], {'mode':'elenco_puntate', 'id': group["id"]})
+    kodiutils.endScript()
+
+def elenco_puntate(id):
+    kodiutils.setContent('videos')
+    for pun in raiyoyo.get_url_punList(id):
+      stamp_ep(pun)
+    kodiutils.endScript()
+
+
+def playRaiyoyo(streamUrl):
+    kodiutils.setResolvedUrl(streamUrl)
```

Comment:
Actually I had some issue with playing on my Mac osx Kodi 16.1 and tried to force using play() ... probably the issue was temporary and then it worked fine ... I also tested on my Krypton kodi on tablet and on Jarvis on tv box and it works fine. 
I forgot to delete the play() ... I will commit changes together with version number change later, thank you.